src/plugins/github/porcelain.test.js

```diff
@@ -40,6 +72,7 @@ describe("GitHub porcelain", () => {
       expect(repo.url()).toBe("https://github.com/sourcecred/example-github");
       expect(repo.owner()).toBe("sourcecred");
       expect(repo.name()).toBe("example-github");
+      expect(deepEqual(repo, asEntity(graph, repo.address()))).toBe(true);
```

Comment:
Could you check whether `expect(x).toEqual(y)` works (in both the positive and negative cases)? If it does, that'd be better—we get a diff in case of failure, rather than just `false`.